_episodes/wrap-up.md

```diff
@@ -129,6 +125,156 @@ Have a look at another example - a simplified directory structure of a blog webs
+
+Once you've clicked that option you will see the box change to say `gh-pages` instead
+of `main` indicating you are working on the new branch you created.
+Creating a `gh-pages` branch also has turned on the website to render at
+`<USERNAME>.github.io/mundana-theme-jekyll`.  You can navigate to `<USERNAME>.github.io/mundana-theme-jekyll`
+and check in "Settings" to see that it worked as expected.
+
+We will be making lots of edits to the new `gh-pages` branch in the challenges below.
+This will be easier to do if we make it the default branch.
+To change the `gh-pages` branch to the default branch, first click 
+the branches button to the right of the drop-down menu we used to create
+the `gh-pages` branch.
+Then in the box that shows the `Default branch` click the two arrows button
+on the upper right-hand side of the box.
+Click that two arrows button again on the next page, choose `gh-pages`
+from the drop down menu options, and click `Update`.
+Confirm your choice by clicking `I understand, update the default branch.`.
+Now when you return to the `Code` tab or the main page of your repo, you will see
+`gh-pages` in the branch drop down by default.
+
+The following series of challenges will have you practice the skills you
+learned in this lesson to modify this template.
+
+> ## Update GitHub About Section Link to Site
+> As we did before in the [Hosting Websites on GitHub episode](github-pages.md), update
+> the repository details to link directly to the rendering website
+> from your fork. 
+> 
+{: .challenge}
+
+> ## Customize the Configuration
+> Update the name and the description variables for the site.
+> Reload the rendered page to see the changes to the actual site.
+> 
+> Hint: See the [Starting with Jekyll episode](starting-jekyll.md) for review
+> 
+{: .challenge}
+
+> ## Customize the Header Menu
+> 
+> Find where the header menu is created and remove the "WP", "Ghost", and 
+> "Buy me a coffee" items.
+>
+> Hint: See the [Reusing Blocks of Content](includes.md) section for more information
+> about where to find the header menu.
+> 
+> > ## Solution
+> > 
+> > Remove lines 12-20 in the `_includes/menu-header.html` file.
+> > 
+> {: .solution}
+{: .challenge}
+
+> ## Update a Layout
+> 
+> NEED SUGGESTION FOR LAYOUT CHANGE 
+>
+> 
+{: .challenge}
+
+> ## Customize the About Page
+>
+> 1. Find the `About` page, update the layout to the `page` layout,
+>  turn off the comments option, remove the image and the "made with" text.
+> 1. Make the page content a reference to the site description you updated
+>  and any other text you'd like to include.
+> 1. Link the `About` page to the header menu and preview it.
+>
+>
+{: .challenge}
+<!--- Should I add directions in the solution? -->
+
+<!--- Challenge:  collection changes - Authors? -->
+> ## Add Yourself as an Author
+>
+> 1. Find where the authors collection is defined and add yourself.
+>
+{: .challenge}
+<!--- Should I add directions in the solution? -->
```

Comment:
Maybe a hint "Start by understanding how to select the author of the first article"

Reply:
Sorry @unode Maybe it has been too long since I drafted this but I'm not quite sure what you mean here?

Reply:
The first time I read this I started by looking at an individual post file but I didn't notice the author name in front matter. GitHub's rendering of made it less visible. So I ended up with:
1. Look for a "author" in `_includes`. Unsuccessful.
2. Look into `_layouts` for some reference to authors. Found `post.html`.
3. Find that `author` is assigned and reference `site.authors` and `page.author`.
4. Confirm that author is mentioned in the page front matter
5. Look into the global site config.

My (clearly *unhintful*) hint above was trying to point to the fact that the author is defined in the front matter, instead of directly in the body.

Would "Hint: How is the author specified in an individual post?" be more helpful?